packages/affiliates/test/mocha/affiliates.js

```diff
@@ -93,12 +95,25 @@ describe("DeployerRewards", function() {
       syntheticTokenDecimals: syntheticTokenDecimals
     });
 
-    assert.equal(Object.keys(result).length, 2); // There should be 2 deplorers for the 3 EMPs.
-    assert.equal(
-      Object.values(result).reduce((total, value) => {
-        return Number(total) + Number(value);
-      }, 0),
-      Number(devRewardsToDistribute)
+    assert.equal(Object.keys(result.deployerPayouts).length, 2); // There should be 2 deplorers for the 3 EMPs.
+    assert.equal(Object.keys(result.empPayouts).length, 3); // There should be 3 emps
+
+    assert.isBelow(
+      // compare floats with an epsilon
+      Math.abs(
+        Object.values(result.deployerPayouts).reduce((total, value) => {
+          return Number(total) + Number(value);
+        }, 0) - Number(devRewardsToDistribute)
+      ),
+      epsilon
```

Comment:
do float check against epsilon because sometimes the math ends up being off by like .000000000001

Reply:
Do you know why this error occurs? Is it because individual payouts are rounded down by BN?

Reply:
I dont know exactly but i think it has to do with floating point math. Theres a division in there and numbers end up being like .9 repeated rather than going evenly into 1.